scripts/index_d_ts

```diff
@@ -466,9 +467,8 @@ declare module "SendbirdUIKitGlobal" {
     onSearchClick?(): void;
     replyType?: ReplyType;
     queries?: ChannelQueries;
-    renderUserProfile?: (props: RenderUserProfileProps) => React.ReactNode | React.ReactElement;
+    renderUserProfile?: (props: RenderUserProfileProps) => React.ReactElement;
     disableUserProfile?: boolean;
-    renderUserMentionItem?: (props: { user: User }) => JSX.Element;
```

Comment:
Why remove this?

Reply:
This is our legacy, we don't get this input on the Channel component